### ISSUE TITLE
Unhandled promise rejection, github-readme-sync@2.1.0

### ISSUE THREAD

**Issue by lfuszara** [readmeio/github-readme-api-sync#40]
```
Run readmeio/github-readme-sync@2.1.0
(node:2496) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'substr' of undefined
    at sanitize (/home/runner/work/_actions/readmeio/github-readme-sync/2.1.0/main.js:12:16)
    at sanitizeKeys (/home/runner/work/_actions/readmeio/github-readme-sync/2.1.0/main.js:36:29)
    at debug (/home/runner/work/_actions/readmeio/github-readme-sync/2.1.0/main.js:43:28)
    at run (/home/runner/work/_actions/readmeio/github-readme-sync/2.1.0/main.js:52:3)
    at Object.<anonymous> (/home/runner/work/_actions/readmeio/github-readme-sync/2.1.0/main.js:174:1)
    at Module._compile (internal/modules/cjs/loader.js:959:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:995:10)
    at Module.load (internal/modules/cjs/loader.js:815:32)
    at Function.Module._load (internal/modules/cjs/loader.js:727:14)
    at Function.Module.runMain (internal/modules/cjs/loader.js:1047:10)
    at internal/main/run_main_module.js:17:11
(node:2496) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:2496) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```
```
name: Sync to ReadMe
on:
  push:
    branches:
      - master

jobs:
  build:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@master
      - uses: readmeio/github-readme-sync@2.1.0
        with:
          readme-oas-key: ${{ secrets.*** }}
```

After update from 2.0 which had bugs, different bug is show.

**Comment by kanadgupta** [comment]
Hi @lfuszara! Thanks for letting us know. This appears to be an issue with the `readme-oas-key` parameter that you've submitted. The format of the key should be `README_API_KEY:API_SPECIFICATION_ID`. Here are our docs on the topic: https://docs.readme.com/docs/automatically-sync-api-specification-with-github

In any case, the error message that you're running into is misleading—apologies for that! Will release a fix for it shortly. Thanks!

**Comment by kanadgupta** [comment]
Fixed in [`v2.1.1`](https://github.com/readmeio/github-readme-sync/releases/tag/2.1.1)!
